the-badass-todo-list-part-1/input.js

```diff
@@ -62,6 +67,7 @@ var TodoInputManager = {
   }
 };
 
+// Quiz#3
```

Comment:
It's fine. Because when I saw some code in real codebase like this I will also spit some f* words. And it's always good to see you can pick such issues.